docs/extending-overseerr/reverse-proxy.md

```diff
@@ -39,6 +39,42 @@ server {
 }
 ```
 
+## Nginx Proxy Manager
+
+Add a new proxy host and configure the following fields:
+
+![NPM example](../../public/images/docs/npm_example.png)
+
+**Tab Details**
+
+1 Your host name for overseerr
+
+2 Keep the scheme on http
+
+3 The internal LAN IP of the server / docker container
+
+4 Port number of the overseerr container (default is 5055)
+
+5 Enable "Cache Assets"
+
+6 Enable “Block Common Exploits”
+
+**Tab SSL**
+
+7 Choose your method for the SSL Certificate. If you are not sure, use “Request a new SSL Certificate”
+
+8 Enable “Force SSL”
+
+9 Enable “HTTP/2 Support”
+
+If you requested a new SSL Certificate in step 7 continue, otherwise click Save
+
+10 Enter your Email address
+
+11 Enable “I Agree to the Let’s Encrypt Terms of Service” if you agree
+
+Click Save
```

Comment:
```suggestion
Add a new proxy host with the following settings:

### Details

- **Domain Names:** Your desired external Overseerr hostname; e.g., `overseerr.example.com`
- **Scheme:** `http`
- **Forward Hostname / IP:** Internal Overseerr hostname or IP
- **Forward Port:** `5055`
- **Cache Assets:** yes
- **Block Common Exploits:** yes

### SSL

- **SSL Certificate:** Select one of the options; if you are not sure, pick “Request a new SSL Certificate”
- **Force SSL:** yes
- **HTTP/2 Support:** yes
```

While nice, I don't think the image is necessary here.  We would also need to update it anytime NPM makes changes to their UI, and I don't think we want to be responsible for maintaining that.

Reply:
Since swag also uses nginx, should we put all nginx solutions into one tab?

In my opinion npm is more than just nginx, exactly like swag is.